namer/marathon/src/test/scala/io/buoyant/namer/marathon/MarathonTest.scala

```diff
@@ -37,6 +40,88 @@ class MarathonTest extends FunSuite {
     assert(marathon.uriPrefix.contains("/marathon"))
     assert(marathon._prefix.contains(Path.read("/io.l5d.marathon")))
     assert(marathon.ttlMs.contains(300))
+    assert(marathon.jitterMs.contains(60))
     assert(!marathon.disabled)
   }
+
+  test("parse config with default jitter") {
+    val yaml = s"""
+                  |kind:           io.l5d.marathon
+                  |prefix:         /io.l5d.marathon
+                  |host:           localhost
+                  |port:           80
+                  |uriPrefix:      /marathon
+                  |ttlMs:          300
+                  |useHealthCheck: false
+      """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(MarathonInitializer)))
+    val marathon = mapper.readValue[NamerConfig](yaml).asInstanceOf[MarathonConfig]
+    assert(marathon.host.contains("localhost"))
+    assert(marathon.port.contains(Port(80)))
+    assert(marathon.uriPrefix.contains("/marathon"))
+    assert(marathon._prefix.contains(Path.read("/io.l5d.marathon")))
+    assert(marathon.ttlMs.contains(300))
+    assert(marathon.jitterMs.isEmpty)
+    assert(!marathon.disabled)
+  }
+
+  test("jitter TTLs with the default jitter") {
+    val yaml = s"""
+                  |kind:           io.l5d.marathon
+                  |prefix:         /io.l5d.marathon
+                  |host:           localhost
+                  |port:           80
+                  |uriPrefix:      /marathon
+                  |ttlMs:          300
+                  |useHealthCheck: false
+      """.stripMargin
+
+    val mapper = Parser.objectMapper(yaml, Iterable(Seq(MarathonInitializer)))
+    val marathon = mapper.readValue[NamerConfig](yaml).asInstanceOf[MarathonConfig]
+    val jitters = Stream.continually(marathon.nextTtl)
+
+    val baseTtl = marathon.ttlMs.get.millis
+    val minTtl = baseTtl - 50.millis
+    val maxTtl = baseTtl + 50.millis
+
+    for (jitter <- jitters.take(300)) {
+      assert(jitter >= minTtl)
+      assert(jitter <= maxTtl)
+    }
+
+    // assert that we're generating a reasonable amount of distinct TTLs
+    assert(jitters.take(300).toSet.size >= 50)
```

Comment:
Yeah, I had this expecting 150 for a while (I hadn't had coffee yet, and instead of doing the math, I thought "half seems reasonable, right?" 😓) and it was failing with lengths around 95-98. 

After I worked out the percentage to actually expect, I was thinking about setting it to 95 but figured I'd lowball it because I didn't want to have a test that breaks occasionally due to probability. I can take it down to 2 but I suspect the odds of it dipping under 50 are pretty low...hmm.